adapters/src/main/java/org/jboss/jca/adapters/jdbc/local/LocalManagedConnectionFactory.java

```diff
@@ -511,19 +506,6 @@ else if (mc.getProperties().equals(newProps))
    }
 
    /**
-    * Is the properties equal
```

Comment:
Lets leave this method in for now - ideally it should be used in equals(). But that requires more extensive testing